src/test/regress/isolation_schedule

```diff
@@ -6,6 +6,7 @@ test: isolation_create_table_vs_add_remove_node
 test: isolation_master_update_node
 test: isolation_ensure_dependency_activate_node
 test: isolation_shouldhaveshards
+test: isolation_alter_role_propagation
```

Comment:
can you move this after `isolation_dump_global_wait_edges ` because  that test's output is changing -- shame on that test being dependent on the previous tests